mesa/visualization/jupyter_viz.py

```diff
@@ -1,3 +1,28 @@
+"""
+Mesa visualization module for creating interactive model visualizations.
+
+This module provides components to create browser-based visualizations of Mesa models,
```

Comment:
While Jupyter is also browser-based, it would be explicit to say both browser-based and Jupyter notebook.

Reply:
Good suggestion, thanks! Added.